src/test/java/selenium/pageobjects/WebLabSubmissionPage.java

```diff
@@ -58,15 +59,22 @@ public void assessWithHints() {
     public void enterSolution(String solution) {
         this.awaitElementVisibility(solutionDiv);
 
-        this.solutionDiv.click();
-        Actions actions = new Actions(driver);
-        actions.keyDown(Keys.CONTROL);
-        actions.sendKeys("a");
-        actions.keyUp(Keys.CONTROL);
-        actions.sendKeys(Keys.BACK_SPACE);
-        actions.sendKeys(solution);
-        actions.perform();
+        String script = String.format("document.getElementById('%s').value = '%s';",
```

Comment:
Same here, you can't do via selenium?

Reply:
I tried set the `textContent` attribute, but couldn't find a way to do that directly via Selenium without the use of JavaScript. Is there a method similar to `getAttribute` but for setting an attribute? It doesn't seem to exist in the `WebElement` class.

Reply:
Hmm google also can't find one. So I guess js magic is required here. Please add a comment explaining that (so that our students, when they read the code, they understand why we did this way)

Reply:
Sure, I added some comments.